frontend/public/components/modals/delete-namespace-modal.tsx

```diff
@@ -46,29 +48,36 @@ export const DeleteNamespaceModal: React.FC<DeleteNamespaceModalProps> = ({
   return (
     <form onSubmit={onSubmit} name="form" className="modal-content ">
       <ModalTitle className="modal-header">
-        <YellowExclamationTriangleIcon className="co-icon-space-r" /> Delete {kind.label}?
+        <YellowExclamationTriangleIcon className="co-icon-space-r" />{' '}
+        {t('public~Delete {{label}}?', { label: kind.label })}
       </ModalTitle>
       <ModalBody>
         <p>
-          This action cannot be undone. It will destroy all pods, services and other objects in the
-          namespace <strong className="co-break-word">{resource.metadata.name}</strong>.
+          {t(
```

Comment:
If possible, this should be one phrase using a Trans component or similar. Breaking things up can make things hard on the translators.

Reply:
This uses the `<Trans>` component and `{resource.metadata.name}` isn't included in the translation. Is this correct?

Reply:
I think it might be better to include the namespace in the translation. The sentence structure we get back from translation is sometimes radically different, so we can't always rely on being able to format the sentence in a given way.

Reply:
Updated